<!-- -->
<launch>
  <node name="camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="camera_frame_id" value="vo_cam" />
    <param name="camera_name" value="DFM42BUC03" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="960" />

    <!--param name="image_width" value="640" />
    <param name="image_height" value="480" /-->
		<!-- in the case of 640 x 480, set 
							v4l2-ctl -d /dev/video1 -c x_offset=320
						  v4l2-ctl -d /dev/video1 -c y_offset=240 -->

    <param name="pixel_format" value="grbg" />
    <param name="bayer_to_grey" value="false"/>
    <param name="io_method" value="mmap"/>
    <param name="field_order" value="none"/>
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="250"/>
    <param name="gain" value="34"/>	
		<param name="camera_info_url" type="string" value="file://$(find usb_cam)/param/dfm_calib.yaml" />		
	</node>
	<!--node name = "image_view" pkg = "image_view" type = "image_view" respawn = "false" output = "screen">
		<remap from = "image" to = "/camera/image_raw" />
		<param name = "autosize" value = "true" />
	</node-->
	<!--node name = "rviz" pkg = "rviz" type = "rviz" respawn = "false" /-->
</launch> 
